src/_includes/templates/nav/primary-nav.html

```diff
@@ -2,7 +2,9 @@
 {% import 'macros/sub-nav.html' as sub_nav %}
 <nav class="primary-nav
             js-primary-nav"
-     aria-expanded="false">
+     aria-expanded="false"
+     aria-label="main navigation"
+     role="navigation">
```

Comment:
I thought it was recommended not to use roles that match the html element, only when the element was something else?